controllers/rhmi/rhmi_controller.go

```diff
@@ -452,41 +452,34 @@ func (r *RHMIReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	if installation.Status.ToVersion == version.GetVersionByType(installation.Spec.Type) && !installInProgress && !productVersionMismatchFound {
 		installation.Status.Version = version.GetVersionByType(installation.Spec.Type)
 		installation.Status.ToVersion = ""
-		metrics.SetRhmiVersions(string(installation.Status.Stage), installation.Status.Version, installation.Status.ToVersion, string(externalClusterId), installation.CreationTimestamp.Unix())
-		if rhmiv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(installation.Spec.Type)) {
-			// remove the code for next release
-			removeOldStages(installation)
-			// end of removal
-			installation.Status.Quota = installationQuota.GetName()
-			installation.Status.ToQuota = ""
-		}
+		metrics.SetVersions(string(installation.Status.Stage), installation.Status.Version, installation.Status.ToVersion, string(externalClusterId), installation.CreationTimestamp.Unix())
+		installation.Status.Quota = installationQuota.GetName()
+		installation.Status.ToQuota = ""
+		// remove the code for next release
+		removeOldStages(installation)
+		// end of removal
 		log.Info("installation completed successfully")
 	}
 
 	// Entered on every reconcile where all stages reported complete
 	if !installInProgress {
 		installation.Status.Stage = rhmiv1alpha1.StageName("complete")
-		metrics.RHMIStatusAvailable.Set(1)
 		retryRequeue.RequeueAfter = 5 * time.Minute
 		if installation.Spec.RebalancePods {
 			r.reconcilePodDistribution(installation)
 		}
 
-		if rhmiv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(installation.Spec.Type)) {
-			if installationQuota.IsUpdated() {
-				installation.Status.Quota = installationQuota.GetName()
-				installation.Status.ToQuota = ""
-				metrics.SetQuota(installation.Status.Quota, installation.Status.ToQuota)
-			}
+		if installationQuota.IsUpdated() {
+			installation.Status.Quota = installationQuota.GetName()
+			installation.Status.ToQuota = ""
+			metrics.SetQuota(installation.Status.Quota, installation.Status.ToQuota)
 		}
 	}
-	metrics.SetRHMIStatus(installation)
+	metrics.SetStatus(installation)
 
-	if rhmiv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(installation.Spec.Type)) {
-		if _, ok := installation.Status.Stages[rhmiv1alpha1.MonitoringStage]; ok {
-			log.Info("delete Monitoring stage from installation.Status")
-			delete(installation.Status.Stages, rhmiv1alpha1.MonitoringStage)
-		}
+	if _, ok := installation.Status.Stages[rhmiv1alpha1.MonitoringStage]; ok {
+		log.Info("delete Monitoring stage from installation.Status")
+		delete(installation.Status.Stages, rhmiv1alpha1.MonitoringStage)
```

Comment:
do we need this?

Reply:
This is part of the move to OO. While I am guess it is no longer needed, it is also related to RHOAM and not RHMI. The removal of RHOAM code is outside the scope of this work.

Reply:
same as above..

Reply:
As this is not a typical PR I do not agree with code changes that are not related to the task at hand. To resolve this I have opened a second PR which has the code removal you are looking for. https://github.com/integr8ly/integreatly-operator/pull/2920

Reply:
Thanks